spec/models/list_import/my_anime_list/row_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.describe ListImport::MyAnimeList::Row do
+  context 'with anime' do
+    let(:anime) { fixture('list_import/my_anime_list/toy-anime.json') }
+
+    describe '#started_at' do
+      context 'with empty date' do
+        it 'should return nil' do
+          subject = described_class.new(JSON.parse(anime)[1])
```

Comment:
You can have separate `subject` within each `context` or `describe` block, they're only applicable within their `do..end` boundary.  The same applies to `let` and `let!` and `before` and `after` and `around`